R/zzz.R

```diff
@@ -0,0 +1,10 @@
+.StattleEnv <- new.env()
```

Comment:
Just one of those weird R traditions. If you need to perform tasks when the package is loaded such as loading libraries you put it in this file. In this case, we're initializing the `.StattleEnv` object. 

More info:
http://stackoverflow.com/questions/21448904/r-packages-what-is-the-file-zzz-r-used-for